app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -340,9 +340,17 @@ public void insertMultipleChoiceCard(MultipleChoiceCard mcCard, SQLiteDatabase d
     }
 
     public void deleteMultipleChoiceCard(MultipleChoiceCard mcCard) {
-        db.delete(MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME,
-                "id" + "=" + mcCard.getId(),
-                null);
+        Cursor res = db.rawQuery("SELECT * FROM " + MultipleChoiceContract.MultipleChoiceCardTable.TABLE_NAME,null);
```

Comment:
Can you explain what the cursor does again please?

Reply:
The cursor contains the result we obtained from the query. When using the cursor the app will have read and write access I believe lol.